k8sdeps/transformer/inventory/inventory.go

```diff
@@ -58,14 +56,20 @@ func NewInventoryTransformer(p *types.Inventory, namespace string, append bool)
 // The prune ConfigMap is used to support the pruning command in the client side tool,
 // which is proposed in https://github.com/kubernetes/enhancements/pull/810
 func (o *inventoryTransformer) Transform(m resmap.ResMap) error {
+	invty := inventory.NewInventory()
 	var keys []string
 	for _, r := range m {
-		s := r.PruneString()
-		keys = append(keys, s)
+		ns, _ := r.GetFieldValue("metadata.namespace")
+		item := resid.New(r.GetGvk(), ns, r.GetName())
```

Comment:
This method should be called `NewItemId` because itemid and resid code are mixed in the same package.

Reply:
updated